### ISSUE TITLE
Landing page image elements do not have [alt] attributes

### ISSUE THREAD

**Issue by rmjordas** [projectunic0rn/mentorship-ui#315]
> Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute. [Learn more](https://dequeuniversity.com/rules/axe/2.2/image-alt?application=lighthouse).

Refer to the [Lighthouse report](https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://www.mentrship.com/) for the list of failing elements.
